mne/decoding/transformer.py

```diff
@@ -61,29 +93,60 @@ def fit(self, epochs_data, y):
         self : instance of Scaler
             Returns the modified instance.
         """
+        from sklearn.base import clone
         if not isinstance(epochs_data, np.ndarray):
             raise ValueError("epochs_data should be of type ndarray (got %s)."
                              % type(epochs_data))
 
         X = np.atleast_3d(epochs_data)
+        n_epochs, n_channels, n_times = X.shape
+        X = X.transpose(1, 0, 2)
+        self.ch_mean_ = np.zeros(n_channels)
+        self.scale_ = np.zeros(n_channels)
+        self.ch_median_ = np.zeros(n_channels)
```

Comment:
? why do you create this variable. You should be able to use ch_mean_ in all cases, should'tn you?